src/app/pages/virtualization/components/all-instances/instance-details/instance-metrics/instance-metrics.component.ts

```diff
@@ -35,6 +35,7 @@ import { ApiService } from 'app/services/websocket/api.service';
 })
 export class InstanceMetricsComponent {
   instance = input.required<VirtualizationInstance>();
+  maxItemsCount = 60 * 10 / 2; // latest 10 minutes with 2 seconds API interval
```

Comment:
I've noticed that `maxItemsCount` is a local variable, however it would be better to make it a constant, to clearly state that it won't be updated in a runtime.